source/issues-and-help.txt

```diff
@@ -0,0 +1,73 @@
+.. _kotlin-sync-issues-and-help:
+
+=============
+Issues & Help
+=============
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: kotlin sync, troubleshooting, feedback
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+We are lucky to have a vibrant MongoDB {+language+} community that includes users
+with varying levels of experience using the {+driver-short+}. The
+quickest way to get support for general questions is through the
+:community-forum:`MongoDB Community Forums </tag/kotlin>`.
+
+Bugs / Feature Requests
+-----------------------
+
+If you have feedback about the {+driver-short+}, visit the `MongoDB
+Feedback Engine <https://feedback.mongodb.com/>`__ and select
+:guilabel:`Drivers` from the list of products on the right side of
+your screen. You can propose improvements, report issues, and provide
+other types of feedback by using this site.
+
+You can also open a case in Jira, our issue management tool, to identify
+bugs or propose improvements. The following steps describe how to create
+a Jira issue:
+
+1. Visit the `MongoDB Jira issue tracker <https://jira.mongodb.org/>`__ and click the 
+   `signup link. <https://account.mongodb.com/account/register>`__
+   Create an account, and then log in to Jira.
+#. Navigate to the `JAVA Jira project. <https://jira.mongodb.org/browse/JAVA>`__
+#. Click :guilabel:`Create` to create a ticket. Please provide as much
+   information as possible about the issue or request in the ticket.
+
+.. note::
+
+   Bug reports in the JAVA Jira project are publicly viewable.
+
+If you’ve identified a security vulnerability in any official MongoDB
+product, please report it according to the instructions found in the
+:manual:`Create a Vulnerability Report page. </tutorial/create-a-vulnerability-report>`
+
+Pull Requests
+-------------
+
+We are happy to accept contributions to help improve the driver. We will guide
+user contributions to ensure they meet the standards of the codebase. Please
+ensure that any pull requests include documentation, tests, and pass the
+**gradle** checks.
+
+To get started check out the source and work on a branch:
+
+.. code-block:: bash
+
+   $ git clone https://github.com/mongodb/mongo-java-driver.git
+   $ cd mongo-java-driver
+   $ git checkout -b myNewFeature
```

Comment:
S: I think you can reword this code intro slightly; also, you can remove the $ from the code example

```suggestion
To get started, clone the source repository and work on a branch by running the following commands:

.. code-block:: bash

   git clone https://github.com/mongodb/mongo-java-driver.git
   cd mongo-java-driver
   git checkout -b myNewFeature
```